rust/enums/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "enums"
```

Comment:
I've been mulling over the naming of each piece, and I think I settled on the idea that directory names shouldn't refer to the programming concept, but rather, the *task* it's trying to achieve. 

I'm in the process of writing the foreword/intro to the book which I'll share with you soon, but in a nutshell, this book is intended to be a top-down approach rather than a bottom-up one. Naming the directory as  concept would mean it's bottom-up (similar to the "Rust by example" book or "The [Rust Reference](https://doc.rust-lang.org/stable/reference/)" book), and mixing up top-down and bottom-up naming just makes it confusing when reading through the side bar index.

So I think for this one, the directories should be named `age_groups_enum`, and organized under the "File-handling" section of the book. Hope that makes sense!

Reply:
Sounds good, I'll rename the directories.

Reply:
Oops, I meant `age_groups_csv` to be consistent with the other directories and to indicate that we're dealing in CSVs.